app/src/config/feature-flags.ts

```diff
@@ -120,7 +120,8 @@ export const launchDarklyService = {
 
 export const fetchFeatureFlag = async (featureFlagKey: string, cloudId?: string): Promise<boolean | null> => {
     try {
-        const environment: Environment = envVars.NODE_ENV as Environment;
+        const environment: Environment = envVars.JENKINS_ENV as Environment || process.env.NODE_ENV;
```

Comment:
I've set JENKINS_ENV in the development, staging, and production environments for the official and outdated apps. Left process.env.NODE_ENV as a fallback as it's needed for tests (locally and in the pipelines).

Reply:
It would be great to add this and the reason we aren't using NODE.ENV as a comment in the codebase